probe/kubernetes/cronjob.go

```diff
@@ -74,3 +86,29 @@ func (cj *cronJob) GetNode() report.Node {
 	}
 	return cj.MetaNode(report.MakeCronJobNodeID(cj.UID())).WithLatests(latest)
 }
+
+func upgrade(legacy *batchv2alpha1.CronJob) *batchv1beta1.CronJob {
```

Comment:
That's too generic a name for a function that lives in the `kubernetes` package. `upgradeCronJob`?